lib/matrix_sdk/request.ex

```diff
@@ -398,6 +595,37 @@ defmodule MatrixSDK.Request do
       headers: [{"Authorization", "Bearer " <> token}]
     }
 
+  @doc """
+  Gets a list of the third party identifiers the homeserver has associated with the user's account.
```

Comment:
Looks you forgot to update this copy and paste in the doc comment.

Reply:
Good catch 👍